remix/remix-solidity/src/compiler/compiler.js

```diff
@@ -702,6 +738,21 @@ function Compiler (handleImportCall, getCompilerAPIUrl) {
 
     cb(null, { 'sources': files, 'target': target })
   }
+
+  function truncateVersion (version) {
+    var tmp = /^(\d+.\d+.\d+)/.exec(version)
+    if (tmp) {
```

Comment:
I just restored this function from its history:

https://github.com/ethereum/remix/blob/master/remix-solidity/src/compiler/compiler.js#L354

So I would suggest we don't change it?

Reply:
I would change it.

I leave the final decision to you, but I'm a great believer in tidying things as you go. Better to be always improving things at the risk of occasionally breaking them, than allowing the codebase to accumulate cruft in the hope that it will stay stable. :-)